src/components/ProgramRecord/RecordsHelp.jsx

```diff
@@ -6,13 +6,13 @@ import { Hyperlink } from '@openedx/paragon';
 function RecordsHelp({ helpUrl }) {
   return (
     <section className="help">
-      <h2>
+      <h3 className="h5">
```

Comment:
Why are you switching to a skipped heading level here?  The previous heading is `h1`.

Reply:
I did it by analogy with this - https://github.com/openedx/frontend-app-learner-record/blob/master/src/components/ProgramCertificatesList/ProgramCertificatesList.jsx#L148 and this - https://github.com/openedx/frontend-app-learner-record/blob/b9265377b31a3188eb3744764ad67855778b8fbe/src/components/ProgramRecordsList/ProgramRecordsList.jsx#L151 files.